src/libraries/System.Runtime.InteropServices/gen/Microsoft.Interop.SourceGeneration/Marshalling/ICustomNativeTypeMarshallingStrategy.cs

```diff
@@ -1051,6 +1067,8 @@ public TypeSyntax AsNativeType(TypePositionInfo info)
         public IEnumerable<StatementSyntax> GenerateCleanupStatements(TypePositionInfo info, StubCodeContext context)
         {
             StatementSyntax contentsCleanupStatements = GenerateContentsMarshallingStatement(info, context,
+                                StubCodeContext.Stage.Cleanup,
+                                null,
```

Comment:
I'd prefer non-nulls when possible. Could we use `StubCodeContext.Stage.Invalid`?